modules/ting_search_carousel/ting_search_carousel.module

```diff
@@ -283,31 +283,34 @@ function _ting_search_carousel_find_entities_with_covers($query, $start, $size,
   $finished = FALSE;
   $entities = array();
 
-  $options = array(
-    'facets' => array(),
-    'enrich' => TRUE,
-    'allObjects' => FALSE,
-  );
+  // Throw an exception if the remainder of the offset divided by the chunk
+  // size is not equal to 0.
+  if (!$start % $size === 0) {
+    throw new InvalidArgumentException('Offset to start from % search chunk size should not have remainders left.');
```

Comment:
I think the exception message could be more precise "Search cffset must be divisible by chunk size".

The current inline comment is basicly a copy/paste from the exception message. In that case we might as well leave it out. It does not provide any additional information. What should be included instead is why we do so: We cannot map a situation where offset is not divisble by chunk size to the search abstraction layer paging model.